core/src/main/scala/cats/syntax/DistributiveSyntax.scala

```diff
@@ -0,0 +1,14 @@
+package cats
+package syntax
+
+import cats.evidence.===
+
+trait DistributiveSyntax extends Distributive.ToDistributiveOps {
+  implicit final def catsSyntaxDistributiveOps[F[_]: Functor, A](fa: F[A]): DistributiveOps[F, A] = new DistributiveOps[F, A](fa)
+}
+
+// Add syntax to functor as part of importing distributive syntax.
+final class DistributiveOps[F[_]: Functor, A](val fa: F[A]) {
```

Comment:
Can we make this an AnyVal? :)

Reply:
No, because the class has the Functor constraint.

Reply:
Ah yes, you're right! We should move the `Functor` constraint to the method instead